Header header

bool isPriorMap

float32 scale
int16[] extentInCells
float32 resolution

float32[] occupancyGrid
float32[] deltaOccupancyGrid